### ISSUE TITLE
Quantitative comparison on DSLR

### ISSUE THREAD

**Issue by Oktai15** [dros1986/content-preserving-tone-adjustment-for-image-enhancement#2]
Hi again, @dros1986!

According to your paper: "training, validation and test sets have respectively 160471,
4304 and 3057 samples". When I downloaded DSLR I have 160471 pairs (iphone, canon) for training and 4353 pairs (iphone, canon) for testing. I downloaded from this site: http://people.ee.ethz.ch/~ihnatova/index.html (`Download patches for CNN training (6.2 GB)` button)

1. Why did you use only 4304 for validation? Or you meant something another?
2. Where can I find 3057 samples for testing?

**Comment by dros1986** [comment]
Hi @Oktai15, sorry for the late answer! I missed the notification. 

The partitions we described in the paper are the ones given during the NTIRE image enhancement competition. In this context, they gave to the participants a validation set of 4304 samples and a testset (without ground-truths) of 3057 images. Later in the challenge, we received a further testset of 10 images, this time at full resolution (1536x2048).

I sent you via email (oktai.tatanov@gmail.com) the validation and test sets.

Please, if you have any further questions, let me know.
